### PR TITLE
Speed up list processing

### PR DESCRIPTION
**Pull request by rocky**
Remove unnecessary evaluations for literal List evaluations.

### CODE REVIEW

File: mathics/eval/nevaluator.py [hunk 1]
```diff
@@ -83,6 +79,21 @@ def eval_NValues(
         result._build_elements_properties()
         return result
 
+    # Get the precision goal to use in non-integer numbers
```
Comment (mmatera) [inline mathics/eval/nevaluator.py:82]: good catch!

**Comment by mmatera** [comment]
@rocky, tell me when this is ready.

**Comment by rocky** [comment]
> @rocky, tell me when this is ready.

It is ready. There will probably be more. But I like small easier-to-follow commits. 

I think there is a way that to_python to pick up (Python) literal values instead of going through the entire list.

**Review by mmatera — APPROVED** [review]
LGTM

**Pull request merged**